inscriptions/ISic000571.xml

```diff
@@ -185,36 +190,40 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138359" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138359" xml:lang="la" resp="#GC">
                 <ab>
                     <lb n="1"/>
```

Comment:
it's much neater if this can all be pulled up onto the same line as the `<lb>` element for each line of text, with words separated simply by a white space, rathe rthan the use of carriage returns.